<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Clavier</class>
 <widget class="QWidget" name="Clavier">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>197</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Centrale d'alarme</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>82</width>
     <height>116</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_1">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_4">
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_7">
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonMarche">
      <property name="text">
       <string>Marche</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>82</width>
     <height>116</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_5">
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_8">
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_0">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>82</width>
     <height>116</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_6">
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_9">
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonArret">
      <property name="text">
       <string>ArrÃªt</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>177</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QCheckBox" name="checkBoxRouge">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Led Rouge</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBoxVerte">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Led verte</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
